bc/subscription/tasks.py

```diff
@@ -53,11 +58,46 @@ def enqueue_posts_for_new_case(
     if document:
         files = get_thumbnails_from_range(document, "[1,2,3,4]")
 
-    for channel in get_channels_per_subscription(subscription.pk):
-        template = get_new_case_template(
-            channel.service, subscription.article_url
+    if initial_document is None:
+        initial_document = lookup_initial_complaint(subscription.cl_docket_id)
+
+    docket: DocketDict | None = None
+    if subscription.cl_docket_id:
+        docket = lookup_docket_by_cl_id(subscription.cl_docket_id)
+
+    date_filed: date | None = None
+    days_old = 0
+
+    date_filed_str = docket["date_filed"] if docket else None
+    if date_filed_str:
+        try:
+            date_filed = (
+                datetime.strptime(date_filed_str, "%Y-%m-%d").date()
+                if date_filed_str
+                else None
+            )
+
+            assert date_filed is not None  # For type checking purposes
+            days_old = (date.today() - date_filed).days
+        except ValueError:
+            # If the date_filed_str is not a valid date, just continue on
+            pass
+
+    initial_complaint_link = (
+        f"https://www.courtlistener.com{initial_document['absolute_url']}"
+        if initial_document
+        else None
+    )
+
+    initial_complaint_type: Literal["Petition", "Complaint"] | None = None
+    if initial_complaint_link and docket:
+        initial_complaint_type = (
+            "Petition" if is_bankruptcy(docket) else "Complaint"
```

Comment:
if we apply the suggestion from https://github.com/freelawproject/bigcases2/pull/530#discussion_r1594384238, we can simplify these lines of code as shown below: 

```python
initial_complaint_type: Literal["Petition", "Complaint"]
initial_complaint_type = (
    "Petition" if is_bankruptcy(subscription.cl_court_id) else "Complaint"
)
```